src/style.scss

```diff
@@ -0,0 +1,87 @@
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 400;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+:root {
+  --shadow-color: 0 3px 8px 0 rgba(61, 78, 97, 0.2);
+  --avenir-font: "avenir", sans-serif;
```

Comment:
It is better to set variable names not by content, but by meaning, for example `--main-font`